deployment/creating&deploying-the-application-using-cli.md

```diff
@@ -1,13 +1,15 @@
-# Deploying the application using CLI
+# AdminJS Cloud CLI for creating and deploing application
 
 AdminJS Cloud Hosting comes with a CLI tool which you can use locally or inside your CI/CD to deploy your application.
 
 ## Installation
 
 {% code overflow="wrap" %}
+
 ```bash
-$ npm i -g @adminjs/cloud-cli
+npm i -g @adminjs/cloud-cli
```

Comment:
Why did you remove `$` at the front? It's standard for documenting bash commands and that's the format we use in the rest of the docs